src/main/java/codechicken/microblock/client/MicroblockRender.java

```diff
@@ -171,7 +170,7 @@ private static void transformFace(Vector3 hit, int side, Matrix4 mat) {
     public static boolean renderCuboids(CCRenderState ccrs, BlockState state, @Nullable RenderType layer, Iterable<MaskedCuboid> cuboids) {
```

Comment:
The boolean return from this can be unwound now. As it has been unwound everywhere else.